src/components/EpisodeList.tsx

```diff
@@ -0,0 +1,17 @@
+import EpisodeView from "./EpisodeView";
+import "../main.css";
+import { IEpisode } from "../IEpisode";
+
+interface SearchedListProps {
+  searchedList: IEpisode[];
```

Comment:
'searchedList' is misleading as not directly linked to the search bar - maybe call it allEpisodes or something referring to episodes for display